app/src/config.py

```diff
@@ -38,6 +38,7 @@ def get_config_env() -> ConfigType | Literal[False]:
             "ta_token": os.environ["TA_TOKEN"],
             "jf_url": os.environ["JF_URL"],
             "jf_token": os.environ["JF_TOKEN"],
+            "jf_folder": os.environ['JF_FOLDER']
```

Comment:
This will raise a KeyError if you don't set that value, as was the default before. So best to implement default and overwrite here in config.py instead of series.py, so we have better separation of concern.